ms2query/ms2library.py

```diff
@@ -262,9 +259,9 @@ def _get_analog_search_scores(self,
                 parent_masses,
                 self.settings["base_nr_mass_similarity"])
 
-            dict_with_preselected_spectra_info[spectrum_id] = results_table
```

Comment:
Descriptive names are good. But wouldn't make them too long and leave out the `with_` part.